### PR TITLE
first version of search for country

### PR DESCRIPTION
**Pull request by Sythrill**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+        <script src="http://www.webglearth.com/v2/api.js"></script>
+        <title>Search engine</title>
+    </head>
+    <body onload="initialize()">
+        <h1>Search for country</h1>
+        <input id="country-name" type="text" onload="avaliableCountries()"/>
+        <button id="search">Search for country</button>
+
```
Comment (matwjablonski) [inline index.html]: empty lines to remove

File: script.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        $('<li><div class="item"><h3>Languages:</h3></div><div class="value">'+item.languages+'</div></li>').appendTo(countriesList);
+        earth.panTo([x,y]);
+        marker = WE.marker([x, y]).addTo(earth);
+    });
+};
+
+var x;
+var y;
+var earth;
+var marker
```
Comment (matwjablonski) [inline script.js]: add `;`

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Sythrill) [reply index.html]: empty lines removed

File: script.js (hunk 2 above)
Comment (Sythrill) [reply script.js]: semicolon on its place

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**